### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by bwegh**
remove the application types, use pure json as it is clear what is returned due to the url .well-known/macaroon

### CODE REVIEW

File: macaroon-minting.xml [hunk 1]
```diff
@@ -85,12 +85,12 @@
...
     <section title="Requesting a macaroon">
       <t>The client requests that the server creates a macaroon by
-      issues an HTTP POST request to the path <spanx
+      issuing an HTTP POST request to the path <spanx
       style="verb">/.well-known/macaroons</spanx>.  The request MUST
       have a <spanx style="verb">Content-Type</spanx> header with
       value <spanx
-      style="verb">application/json+macaroon-request</spanx>.  The
-      request entity must be a valid JSON object.</t>
+      style="verb">application/json</spanx>.  The request entity
```
Comment (paulmillar) [inline macaroon-minting.xml:92]: Why not "application/json+macaroon-request" ?

File: macaroon-minting.xml [hunk 2]
```diff
@@ -52,7 +52,7 @@
       was issued.</t>
 
       <t>This document describes a simple API that allows a client to
-      request a macaroon be minted that captures the client's
+      request a macaroon to be minted that encodes the client's
```
Comment (paulmillar) [inline macaroon-minting.xml:55]: I'm not sure the extra "to" is needed (or even is correct) here, but I can rephrase the sentence if it is confusing.

File: macaroon-minting.xml (hunk 1 above)
Comment (bwegh) [inline macaroon-minting.xml:92]: if it should be "application/json;macaroon-reguest' im my oppinion
https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.7

File: macaroon-minting.xml (hunk 2 above)
Comment (bwegh) [inline macaroon-minting.xml:55]: neither do I know, yet it sounded strange to me and with the 'to' it sounds way better to me ...

File: macaroon-minting.xml (hunk 1 above)
Comment (paulmillar) [inline macaroon-minting.xml:92]: OK, that would work too.

Another alternative would be to use "/.well-known/macaroons/mint" as the endpoint and simply "application/json" as the Content-Type.  The caveat discovery service would use a different endpoint ("/.well-known/macaroons/caveats" for example).

Comment (paulmillar) [inline macaroon-minting.xml:92]: Do you want to chat more on IDC-AAI slack?  I think that's a better tool for a conversation when discussing this further.

Comment (bwegh) [inline macaroon-minting.xml:92]: sure I am on there

**Pull request merged**